udata_front/theme/gouvfr/templates/raw.html

```diff
@@ -12,7 +12,6 @@
     {% block extra_css %}{% endblock %}
     {% block theme_css %}
     <link href="{{ theme_static('assets/style.css') }}" rel="stylesheet">
-    <link href="{{ theme_static('assets/index.css') }}" rel="stylesheet">
```

Comment:
:eyes: Have we had this broken css import for years?

Reply:
No, this css file is now merged in `style.css` since the changes to the way we import datagouv/components. We added `cssCodeSplit: false` to the build so this file was removed